static/main.js

```diff
@@ -1,30 +0,0 @@
-document.body.className = document.body.className
-  ? document.body.className + " js-enabled"
-  : "js-enabled";
-
-window.GOVUKFrontend.initAll();
-
-window.onload = () => {
```

Comment:
We still need this script, it's just this `onload` that should go away.

Reply:
😖Ahhh sorry!